ios/FluentUI/List/ListItem.swift

```diff
@@ -436,3 +433,10 @@ public extension ListItem where LeadingContent == EmptyView, TrailingContent ==
         }
     }
 }
+
+public extension ListItem {
+    /// Provide override values for various `ListItem` values.
+    mutating func overrideTokens(_ overrides: [ListItemToken: ControlTokenValue]) {
```

Comment:
This function needs to return a type of `Self` and look like this:
```swift
mutating func overrideTokens(_ overrides: [ListItemToken: ControlTokenValue]) {
    tokenOverrides = overrides
    return self
}
```
That way, it can be chained with other modifiers (like your other comment mentions).

Reply:
ah yes this! I knew I was missing something obvious. thanks Mike!